common/views/components/BasicBody/BasicBody.js

```diff
@@ -21,7 +24,19 @@ const BasicBody = ({ body }: Props) => {
     <div className='basic-body'>
       {body.map((slice, i) =>
         <div className={`body-part ${spacing({s: 4}, {margin: ['top']})}`} key={`slice${i}`}>
-          {slice.type === 'text' && <div className='body-text'><HTMLBlock html={slice.value} /></div>}
+          {slice.type === 'text' &&
+            <div className={`
```

Comment:
Would there be any value in updating the Standfirst component to be something like a FeaturedText component and choosing between that and the HTMLBlock component based on the slice weight?